source/Government.cpp

```diff
@@ -657,7 +665,10 @@ bool Government::Condemns(const Outfit *outfit) const
 {
 	const auto isAtrocity = atrocityOutfits.find(outfit);
 	bool found = isAtrocity != atrocityOutfits.cend();
-	return (found && isAtrocity->second) || (!found && outfit->Get("atrocity") > 0.);
+	if(found)
+		return isAtrocity->second;
+	else
+		return !ignoreUniversalAtrocities && outfit->Get("atrocity") > 0.;
```

Comment:
```suggestion
		return !IgnoresUniversalAtrocities() && outfit->Get("atrocity") > 0.;
```
All these ignoreUniversalAtrocities should use the member function instead.